### ISSUE TITLE
Shell should be retargeted

### ISSUE THREAD

**Issue by Krutonium** [IronManMark20/ShellSharp#15]
Right now it's targeted for .net 2.0 - Which does not ship be default on Windows 8 and newer. It compiles fine under far newer .net 4.

**Comment by emmatyping** [comment]
Really? I thought I changed that. I'll retarget. I've been testing on 4.6

**Comment by Krutonium** [comment]
It's .net 2 by default, it prompted me to change it on a fresh clone since I don't have it installed on this system.

**Comment by emmatyping** [comment]
Uh, `<TargetFrameworkVersion>v4.6</TargetFrameworkVersion>` in all the csproj files I checked.

**Comment by Krutonium** [comment]
>.>

Lemme recheck.

**Comment by Krutonium** [comment]
![image](https://cloud.githubusercontent.com/assets/3945538/11455098/bc384bb0-9619-11e5-8890-763241c21008.png)

**Comment by emmatyping** [comment]
Ah. Try re-cloning and run the .bat

**Comment by Krutonium** [comment]
Did you change it in the last 4 minutes?

**Comment by emmatyping** [comment]
Yes https://github.com/IronManMark20/gong-shell/commit/50b49de44f4a5356e2b65e3dffd0606854bd16e0

**Comment by Krutonium** [comment]
Ok

**Comment by Krutonium** [comment]
![image](https://cloud.githubusercontent.com/assets/3945538/11455128/21c3a5f0-961b-11e5-91ff-a7042d2f769d.png)

**Comment by Krutonium** [comment]
It's still happening

**Comment by emmatyping** [comment]
Hmm. try from the command line `git submodule foreach git pull origin master`

**Comment by emmatyping** [comment]
Or `git submodule update --recursive`

**Comment by Krutonium** [comment]
Only update was for Explore10

**Comment by emmatyping** [comment]
Okay, try again, clone recursive, no need for the bat (I might remove it actually).
